_docs/_latest/setup/upgrade.md

```diff
@@ -5,71 +5,44 @@ order: 00
 
 # {{ page.title }}
 
-This guide explains how to use the Forseti upgrade tool.
+This guide explains how to upgrade your Forseti from v1.x to v2.
 
-## Before you begin
+## Important notes
 
-Before you upgrade Forseti, you'll need the following:
+ * We currently don't support data migration from v1 to v2, so you will need to archive the database manually 
```

Comment:
remove the use of the word currently as we don't have any intention so this won't change.

Reply:
Updated.